Dealership Homework/Dealership/src/main/java/ro/siit/ElectricVehicle.java

```diff
@@ -0,0 +1,62 @@
+package ro.siit;
+
+/**
+ * Created by Dani on 25-Aug-17.
+ */
+public class ElectricVehicle extends Manufacturer {
```

Comment:
ElectricVehicle has-a manufacturer - true
ElectricVehicle is-a manufacturer - false